### PR TITLE
Fix/wizard

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Add support for user managing accounts with the API
MBE-104
### Description of the Change

If the users manages its accounts with the API, he or she may not have any accounts left for the Wizard, even though this should not append

### CODE REVIEW

File: src/exchange/wizard-hosted-creation/wizard-hosted-creation.component.js [hunk 1]
```diff
@@ -7,27 +7,48 @@
       messaging,
       navigation,
       ovhUserPref,
+      wizardHostedCreationDomainConfiguration,
+      wizardHostedCreationEmailCreation,
+      $q,
```
Comment (antleblanc) [inline src/exchange/wizard-hosted-creation/wizard-hosted-creation.component.js]: Could be sorted? :)

File: src/exchange/header/header.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            </div>
+            <div data-ng-if="!$ctrl.isSubmittingNewDisplayName">
+                <div
+                    data-ng-if="!$ctrl.isEditingDisplayName"
+                    class="d-flex align-items-center">
+                    <h1
+                        class="my-0"
+                        data-ng-bind="$ctrl.remoteDisplayName"></h1>
+                    <button
+                        class="btn btn-icon ml-4"
```
Comment (marie-j) [inline src/exchange/header/header.html]:
```suggestion
                        class="oui-button oui-button_icon-only ml-4"
```

File: src/exchange/header/header.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                    data-ng-bind="$ctrl.exchangeService.domain"
+                    data-ng-if="$ctrl.exchangeService.domain !== $ctrl.exchangeService.displayName"></span>
+            </div>
+        </div>
+        <oui-action-menu
+            class="col-sm-offset-1 col-sm-1"
+            data-text="{{:: 'common_actions' | translate }}">
+            <oui-action-menu-item
+                data-ng-if="$ctrl.canSubscribeToSharepoint"
+                href="#/configuration/sharepoint/activate/{{:: $ctrl.exchangeService.organization }}/{{:: $ctrl.exchangeService.domain }}">
```
Comment (marie-j) [inline src/exchange/header/header.html]: No `data-` ?

File: src/exchange/header/header.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+                <span data-translate="{{:: 'exchange_tab_INFORMATIONS_order_sharepoint' | translate }}"></span>
```
Comment (marie-j) [inline src/exchange/header/header.html]:
```suggestion
                <span data-translate="'exchange_tab_INFORMATIONS_order_sharepoint"></span>
```

File: src/exchange/header/header.html [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+            </oui-action-menu-item>
+            <oui-action-menu-item
+                data-ng-if="$ctrl.canSubscribeToOfficeAttach"
+                href="#/configuration/microsoft/office/license/order">
+                <span data-translate="{{:: 'exchange_tab_INFORMATIONS_order_office' | translate }}"></span>
```
Comment (marie-j) [inline src/exchange/header/header.html]:
```suggestion
                <span data-translate="exchange_tab_INFORMATIONS_order_office"></span>
```

File: src/exchange/header/header.html [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+            </oui-action-menu-item>
+            <oui-action-menu-item
+                data-ng-if="$ctrl.canSubscribeToOfficeAttach"
+                data-on-click="$ctrl.navigation.setAction('exchange/office-attach/dialog/office-attach-dialog', $ctrl.exchangeService)">
+                <span data-translate="{{:: 'exchange_ACTION_export_accounts_office_attached_button' | translate }}"></span>
```
Comment (marie-j) [inline src/exchange/header/header.html]:
```suggestion
                <span data-translate="exchange_ACTION_export_accounts_office_attached_button}"></span>
```

File: src/exchange/header/header.html [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+            </oui-action-menu-item>
+            <oui-action-menu-divider data-ng-if="$ctrl.canSubscribeToSharepoint || $ctrl.canSubscribeToOfficeAttach"></oui-action-menu-divider>
+
+            <oui-action-menu-item
+                data-on-click="$ctrl.navigation.setAction('exchange/header/license/service-license-history', $ctrl.exchangeService)">
+                <span data-translate="{{:: 'exchange_action_license_history_button' | translate }}"></span>
```
Comment (marie-j) [inline src/exchange/header/header.html]:
```suggestion
                <span data-translate="exchange_action_license_history_button></span>
```

File: src/exchange/header/header.html [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+            </oui-action-menu-item>
+            <oui-action-menu-item
+                data-ng-if="$ctrl.exchangeService.renewOptionAvailable"
+                data-on-click="$ctrl.navigation.setAction('exchange/header/update-renew/update-renew', $ctrl.exchangeService)">
+                <span data-translate="{{:: 'exchange_update_billing_button_title' | translate }}"></span>
```
Comment (marie-j) [inline src/exchange/header/header.html]:
```suggestion
                <span data-translate="exchange_update_billing_button_title"></span>
```

File: src/exchange/header/header.html [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+            </oui-action-menu-item>
+
+            <oui-action-menu-divider data-ng-if="$ctrl.exchangeService.deleteOptionAvailable"></oui-action-menu-divider>
+            <oui-action-menu-item
+                data-ng-if="$ctrl.exchangeService.deleteOptionAvailable"
+                data-on-click="$ctrl.navigation.setAction('exchange/header/remove/exchange-remove', $ctrl.exchangeService)">
+                <span
+                    data-ng-if="$ctrl.exchangeService.renewType.deleteAtExpiration"
+                    data-translate="{{:: 'exchange_resilitation_action_button_cancel' | translate }}"></span>
```
Comment (marie-j) [inline src/exchange/header/header.html]:
```suggestion
                    data-translate="exchange_resilitation_action_button_cancel"></span>
```

File: src/exchange/header/header.html [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+                <span
+                    data-ng-if="!$ctrl.exchangeService.renewType.deleteAtExpiration"
+                    data-translate="{{:: 'exchange_resilitation_action_button' | translate }}"></span>
```
Comment (marie-j) [inline src/exchange/header/header.html]:
```suggestion
                    data-translate="exchange_resilitation_action_button"></span>
```

File: src/exchange/header/header.html (hunk 3 above)
Comment (FredericEspiau) [reply src/exchange/header/header.html]: No, this is valid `HTML5` and valid according to our convention :)

**Pull request merged**